katas/content/superposition/hardy_state/solution.md

```diff
@@ -0,0 +1,19 @@
+> This solution is a detailed explanation of [this StackExchange answer](https://quantumcomputing.stackexchange.com/a/6844/2879).
+
```

Comment:
Let's drop this link, that answer was written by me so nobody will get insulted by us not giving them credit, and the explanation here is enough for the learner
```suggestion
```